Assets/Scripts/UI/DialogBox/Options/DialogBoxOptions.cs

```diff
@@ -36,7 +36,7 @@ public void OnButtonOpenSettings()
     {
         this.CloseDialog();
         dialogManager.dialogBoxSettings.ShowDialog();
-    }
+	}
```

Comment:
I don't think your editor is configured correctly since your using tabs. you could manually replace tabs with spaces, but its easier to change your monodevelop settings by looking at style guildines in the [contributing file](https://github.com/TeamPorcupine/ProjectPorcupine/blob/master/CONTRIBUTING.md#style-guidelines)